### ISSUE TITLE
[Request] portable builds for Windows

### ISSUE THREAD

**Issue by ItsCubeTime** [colesbury/nogil#129]
Would like to try out nogil without having to use the installer

**Comment by ApexArtist** [comment]
same

**Comment by colesbury** [comment]
Sorry, I don't think I'll have time to make portable builds for Windows. I'm currently focused on getting these changes merged upstream in the upcoming CPython 3.13 release.

**Comment by ApexArtist** [comment]
> Sorry, I don't think I'll have time to make portable builds for Windows. I'm currently focused on getting these changes merged upstream in the upcoming CPython 3.13 release.

no problem.. wen release ?

**Comment by colesbury** [comment]
https://peps.python.org/pep-0719/

**Comment by stonebig** [comment]
Can we hope  a first  free-thread build for 3.13.0a3 ? Or is it too optimistic?

**Comment by ApexArtist** [comment]
excited

**Comment by colesbury** [comment]
No, the goal is to have the major components integrated by the first beta and in a good state with the first release candidate.

**Comment by ovo-Tim** [comment]
Would you like to try conda? `conda install ovo-tim::nogil`
